services_app/src/main/java/org/opendatakit/services/forms/provider/FormsProvider.java

```diff
@@ -56,7 +56,7 @@
  * forms within the ODK toolsuite.
  */
 public class FormsProvider extends ContentProvider {
-  static final String t = "FormsProvider";
+  static final String providerID = "FormsProvider";
```

Comment:
Please use something like this:

private static final String LOGTAG = FormsProvider.class.getSimpleName();